e2e-tests/test-all.sh

```
@@ -1,12 +1,19 @@
-#!/bin/sh
+#!/usr/bin/env bash
 
 set -eu
 
+cd "$(dirname "$0")"
+if [[ -z "${1:-}" ]]; then FILTER="*"; else FILTER="*$1*"; fi
+
 for x in *; do
     if test -d "$x"; then
+			if [[ "$x" != $FILTER ]]; then
```

Shellcheck (0.11.0)

[warning] Quote the right-hand side of != in [[ ]] to prevent glob matching.

(SC2053)